odoo/_monkeypatches/num2words.py

```diff
@@ -1,10 +1,13 @@
 import decimal
+import logging
 import math
 import re
 from collections import OrderedDict
 from decimal import ROUND_HALF_UP, Decimal
 from math import floor
 
+import odoo
+
```

Comment:
We both know that re-importing odoo here is fine because it only uses `MIN_PY_VERSION` which exists inside the module already. Nevertheless I prefer to explicitely *only* import that symbol
```suggestion
from odoo import MIN_PY_VERSION
```